man/fi_domain.3.md

```diff
@@ -76,11 +76,14 @@ domain interfaces are outside the scope of this documentation.
 
 ## fi_domain_bind
 
-Associates an event queue with the domain.  An event queue bound to a
-domain will be the default EQ associated with asynchronous control
-events that occur on the domain or active endpoints allocated on a
-domain.  This includes CM events.  Endpoints may direct their control
-events to alternate EQs by binding directly with the EQ.
+Associates an event queue with a fid.
+
+Currently, only domains can be associated with an event queue.  An
```

Comment:
The intent is that event queues can be associated with passive endpoints (for CM events), active endpoints (for CM events), and address vectors (for insertion events).  An EQ can be associated with a domain, which makes the EQ the default EQ for all EPs and AVs.